src/options/converge.ts

```diff
@@ -31,6 +31,8 @@ export function convergeOptions(options: ParserOptions & PugParserOptions): PugP
 		pugSortAttributesEnd: options.pugSortAttributesEnd,
 		pugWrapAttributesThreshold: options.pugWrapAttributesThreshold,
 		pugWrapAttributesPattern: options.pugWrapAttributesPattern,
+		pugIdNotation: options.pugIdNotation,
+		pugClassNotation: options.pugClassNotation,
```

Comment:
mh, just a minor thing, but everywhere it seams we have class then id
so we could switch these two line ^^'